android/build.gradle

```diff
@@ -66,32 +55,26 @@ allprojects {
     }
 
     repositories {
-        exclusiveContent {
-           // We get React Native's Android binaries exclusively through npm,
-           // from a local Maven repo inside node_modules/react-native/.
-           // (The use of exclusiveContent prevents looking elsewhere like Maven Central
-           // and potentially getting a wrong version.)
-           filter {
-               includeGroup "com.facebook.react"
-           }
-           forRepository {
-               maven {
-                   url "$rootDir/../node_modules/react-native/android"
-               }
-           }
-        }
-        mavenLocal()
-        // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        maven { url("$rootDir/../node_modules/react-native/android") }
+//         exclusiveContent {
+//                    filter {
+//                        includeGroup "com.facebook.react"
+//                    }
+//                    forRepository {
+//                        maven {
+//                            url "$rootDir/../node_modules/react-native/reactAndroid"
+//                        }
+//                    }
+//         }
+//         mavenLocal()
         // Android JSC is installed from npm
         maven { url("$rootDir/../node_modules/jsc-android/dist") }
         // For geth, function, and status-go
         flatDir { dirs "libs", System.env.STATUS_GO_ANDROID_LIBDIR }
         // Everything Else
-        google()
-        jcenter() // Required for @react-native-community/blur:3.6.1, can be removed after updating to 3.6.1+
-        mavenCentral()
+//         google()
+//         jcenter() // Required for @react-native-community/blur:3.6.1, can be removed after updating to 3.6.1+
+//         mavenCentral()
         //         maven { url "https://www.jitpack.io" }
-        maven { url 'https://mvnrepository.com/artifact/com.github.gundy/semver4j'}
+//         maven { url 'https://mvnrepository.com/artifact/com.github.gundy/semver4j'}
     }
 }
```

Comment:
This doesn't look right to me. What was the reason of removing?
Fixing some build issues?

Reply:
This is for debug purpose to try to fix an issue, which id on CI/build-android currently.